circuits/types/integers/src/compare.rs

```diff
@@ -88,6 +87,9 @@ mod tests {
         num_private: usize,
         num_constraints: usize,
     ) {
+        // Note that we need to use "fresh" circuits for each of the test cases, otherwise number of
+        // variables and constraints are incorrectly counted.
+
```

Comment:
What is this comment addressing specifically?

Circuit::scope creates a new environment each time

Reply:
Removing comment.